.github/CODEOWNERS

```diff
@@ -1,2 +1,2 @@
 # Main repo owners:
-*       @dfunckt
+*       @dfunckt @xginn8 @pdcastro
```

Comment:
Is this like an "honorary degree" that some universities award to people because they are famous? With the difference that I am not famous. 😁  I did get as far as installing openBalena and testing the CLI a few times though.

Reply:
I thought your product/user-centric perspective will be extremely useful :)